packages/server/src/core/internals/config.ts

```diff
@@ -12,13 +12,33 @@ export interface InitGenerics {
   ctx: Record<string, unknown>;
   meta: Record<string, unknown>;
 }
+
+/**
+ * The default check to see if we're in a server
+ */
+export const isServerDefault: boolean =
+  typeof window === 'undefined' ||
+  !('Deno' in window) ||
+  process.env.NODE_ENV === 'test' ||
+  !!process.env.JEST_WORKER_ID;
+
 /**
  * The initial params that are used and actually represents real values underneath
  * @internal
  */
 export interface InitOptions<TType extends InitGenerics> {
   transformer: DataTransformerOptions;
   errorFormatter: ErrorFormatter<TType['ctx'], any>;
+  /**
+   * Allow `@trpc/server` to run in non-server environments
+   * @default false
+   */
+  allowOutsideOfServer: boolean;
+  /**
+   * Is this a server environment?
+   * @default typeof window !== 'undefined'
+   */
+  isServer: boolean;
```

Comment:
This option doesn't make sense to me. The default is true when it's in a browser environment, not a server environment.

Reply:
Fixed!